capellambse/_diagram_cache.py

```diff
@@ -62,6 +62,7 @@ def export(
     format: str,
     index: bool,
     force: t.Literal["exe", "docker", None],
+    background: t.Optional[bool],
```

Comment:
There's no difference between the "False" and the "None" case, so there's no need to distinguish between them.

```diff
-    background: t.Optional[bool],
+    background: bool,
```